.lintstagedrc.json

```
@@ -0,0 +1,3 @@
+{
+    "*.js": "eslint --fix"
```

```suggestion
    "*.{js,cjs}": "eslint --fix"
```

We have a few `.cjs` files too.